features/car/conditional_restrictions.feature

```diff
@@ -538,9 +538,9 @@ Feature: Car - Turn restrictions
 
         When I route I should get
             | from | to | route                                      | turns                                       |
-            | a    | e  | cap south,florida nw,florida nw,florida ne | depart,turn right,continue uturn,arrive     |
+            | a    | e  | cap south,florida ne,florida ne | depart,turn left,arrive     |
```

Comment:
I am curious how it is possible that it was changed due to changes in JSON rendering? 🤔

Reply:
I guess it is due to some caches on CI because you have new failure which says you have "original" values here 
https://pipelines.actions.githubusercontent.com/serviceHosts/5f8b3a21-777f-4ccd-83b2-9ebc854aa1e0/_apis/pipelines/1/runs/1148/signedlogcontent/31?urlExpires=2023-02-02T09%3A15%3A35.9673538Z&urlSigningMethod=HMACV1&urlSignature=k5UH1girtk3krsR2TdceIF1EsrnxhL2RtXruekV8s%2BU%3D

Reply:
This case is torturing me non-stop hahahaha, do you have a good solution?

Reply:
Yes, I think I can try dropping CI caches. Sorry, I am a bit busy right now, but will try to do that this evening(not sure which time zone you are, it will be in ~8-10 hours).

Reply:
> 

Then this issue will be handed over to you, and I will no longer submit relevant code. I'm in GMT+8